src/apps/migrate/services/mongo.py

```diff
@@ -1356,6 +1379,120 @@ def get_anons(self, anon_id: uuid.UUID) -> List[AppletUserDAO]:
             )
         return res
 
+    def get_roles_mapping_from_applet_profile(
+        self, migrated_applet_ids: List[ObjectId]
+    ):
+        applet_collection = self.db["folder"]
+        not_found_users = []
+        access_result = []
+        applet_profiles = self.db["appletProfile"].find(
+            {
+                "appletId": {"$in": migrated_applet_ids},
+                "roles": {"$exists": -1},
+            }
+        )
+
+        owner_count = 0
+        manager_count = 0
+        reviewer_count = 0
+        editor_count = 0
+        coordinator_count = 0
+        respondent_count = 0
+        managerial_applets = []
+
+        for applet_profile in applet_profiles:
+            if applet_profile["userId"] in not_found_users:
```

Comment:
not_found_users is declared, and checked but not populated. is it on purpose?

Reply:
fixed